_posts/2017-09-21-Anna-story.markdown

```diff
@@ -0,0 +1,74 @@
+My story begins at the point where I started **studying computer science without any prior programming knowledge.**
```

Comment:
@Anialeinchen Can you add prior to this line 6 lines with: layout, title, date and categories and separators?

Here is an example:
https://github.com/WTMBerlin/wtmberlin.github.io/pull/10/files#diff-bf44a554a72fc8deb628cfbdc4442625R2